src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -3213,36 +3213,44 @@ void Unit::_UpdateSpells(uint32 time)
 
 void Unit::_UpdateAutoRepeatSpell()
 {
-    // check "realtime" interrupts
-    if ((GetTypeId() == TYPEID_PLAYER && ToPlayer()->isMoving()) || IsNonMeleeSpellCast(false, false, true, m_currentSpells[CURRENT_AUTOREPEAT_SPELL]->m_spellInfo->Id == 75))
+    SpellInfo const* spellProto = nullptr;
+    if (m_currentSpells[CURRENT_AUTOREPEAT_SPELL])
+        spellProto = m_currentSpells[CURRENT_AUTOREPEAT_SPELL]->m_spellInfo;
+
+    if (!spellProto)
+        return;
+
+    // Check "realtime" interrupts
+    if ((GetTypeId() == TYPEID_PLAYER && ToPlayer()->isMoving() && spellProto->Id != 75) || IsNonMeleeSpellCast(false, false, true, spellProto->Id == 75))
```

Comment:
it would be nice to avoid magic numbers like "75" and wrap them into constants instead

Reply:
Where do you want me to put it exactly? In this function or to move it to some other file?

Reply:
@FrancescoBorzi maybe something like this?

![Screenshot_8](https://user-images.githubusercontent.com/8354705/97112733-3573c200-16e6-11eb-9fe1-81b3658c1070.png)